package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^7.32.0",
```

Comment:
- [ ] It is one of the project requirements for the book Data to be preserved in the browser's memory by using [localStorage](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md), then reload the page manually to check this after adding a book and removing a book. however everytime I reloard the page. all the date get delete from the local storage. Please address this 💯 👍🏻